example/accel/offload-template/README.md

```diff
@@ -0,0 +1,48 @@
+Minimal Geant4-Celeritas offloading template
+============================================
```

Comment:
Can you make this file an RST and include it using
```rst
.. include:: ../../example/offload-template/README.rst
```
in the `doc/example/geant4.rst` ?